Src/Clients/Core/Common/Models/Storage/Genres/GenreBindingModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5384135)